prow/generate-transform-jobs/main_test.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2019 Istio Authors
```

Comment:
Nit: the header is inconsistent with others.

Reply:
Thanks. Let me fix that one. At some point, we should probably go through the rest of this repo and fix the other copyrights similar to https://github.com/istio/istio/pull/23958.